docs/docker-integration/manual-install.rst

```diff
@@ -0,0 +1,67 @@
+.. _installing-flocker-with-docker:
+
+=============================================
+Manually Installing Flocker with Docker Swarm
```

Comment:
This page's parent talks about installing Flocker with Docker Swarm and Compose. So, user might expect link to installing Docker Compose in this page? Thanks.